### PR TITLE
 WIP: /storages and /storageDomains branches of store merged 

### PR DESCRIPTION
**Pull request by jniederm**
Only the last commit is relevant, the rest is to express dependency on #627.

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/633)

### CODE REVIEW

File: src/reducers/hosts.js [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-    })
-    const imUpdates = fromJS(updates)
-    return state.mergeIn(['hosts'], imUpdates)
-  },
+const initialState = fromJS({})
 
-  REMOVE_MISSING_HOSTS (state, { payload: { hostIdsToPreserve } }) {
-    return removeMissingItems({ state, subStateName: 'hosts', idsToPreserve: hostIdsToPreserve })
+const hosts = actionReducer(initialState, {
+  [SET_HOSTS] (state, { payload: hosts }) {
```
[the remark this answers is not in the document]
Comment (jniederm) [reply src/reducers/hosts.js]: Is the events patch close to merge? If yes, I'll rework this follow add/remove semantics.

**Comment by mareklibra** [comment]
Can you please rebase? Otherwise ready to merge

**Comment by jniederm** [comment]
rebased

**Comment by mareklibra** [comment]
Thanks

**Pull request merged**